serialver: add page (#9371)
tldr-pages/tldr: pages/linux/serialver.md

# serialver

> Returns the serialVersionUID of classes.
> It does not set a security manager by default.

> > More information: <https://manned.org/serialver>.

Reviewer: ```suggestion
> More information: <https://docs.oracle.com/en/java/javase/18/docs/specs/man/serialver.html>.
```
Update the link to official documentation, also this page must be placed in `common`.

# serialver

> Returns the serialVersionUID of classes.
> It does not set a security manager by default.

> > More information: <https://manned.org/serialver.1>.

Reviewer 2: ```suggestion
> More information: <https://manned.org/serialver>.
```

> - Display serialVersionUID of a class:

Reviewer 2: ```suggestion
- Display the serialVersionUID of a class:
```

`serialver {{classnames}}`

> - Display the user interface, enter full class name and press show to view the serial version:

Reviewer 2: Is this an interactive example?

Author: yes

> `serialver -show {{classnames}}`

Reviewer 2: I checked the command and this `-show` flag is not available in the latest version. Just remote it.

Author: done

- Display serialVersionUID of a class in a specific package, separate classes and resources with a colon(:):

`serialver -classpath {{directory}} {{classnames}}`

> - Pass specified option from reference page of Java application launcher to the Java Virtual Machine:

Reviewer 2: ```suggestion
- Use a specific option from reference page of Java application launcher to the Java Virtual Machine:
```

> - Pass specified option from reference page of Java application launcher to  the Java Virtual Machine:

Reviewer 2: ```suggestion
- Pass specified option from reference page of Java application launcher to the Java Virtual Machine:
```
